concepticondata/ignore.tsv

```diff
@@ -0,0 +1 @@
+ID	GLOSS	COMMENT
```

Comment:
Just did my final check and noticed that the entry in the `ignore.tsv` file is no longer there. @martino-vic Could you add it back?

Reply:
I removed its content as a result of this https://github.com/concepticon/concepticon-data/pull/1240#discussion_r995615044 discussion. I left the file there though, in case somebody in the future wants to fill it up, so it's already there.

Reply:
I see!